app/models/brand.rb

```diff
@@ -182,8 +184,47 @@ def in_trial?
     !!subscription.try(:trial?)
   end
 
+  def delete_account
```

Comment:
@signalplus-dev ah, yea might be easier just deleting all. I did already do the reviving accounts though.